packages/@azure/servicebus/data-plane/test/sendSchedule.spec.ts

```diff
@@ -298,12 +298,12 @@ describe("Schedule multiple messages to Queue/Subscription", function(): void {
     should.equal(
       msgEnqueueTime1 - scheduleTime.valueOf() >= 0,
       true,
-      "msgEnqueueTime is different than expected"
+      "Enqueued time must be greater than scheduled time"
     );
     should.equal(
       msgEnqueueTime2 - scheduleTime.valueOf() >= 0,
       true,
-      "msgEnqueueTime is different than expected"
+      "Enqueued time must be greater than scheduled time"
```

Comment:
When there are multiple such messages in same test it's definitely hard to pinpoint which one's throwing it. I would update this to differentiate the messages in same test, somehow.

Reply:
In fact, I like the previous use of variable reference in message (would like that to be the case for all messages if possible, but can skip if that's too much for now)
Here, would use msgEnqueueTime1 and msgEnqueueTime2 to differentiate the msgs.

Reply:
Makes sense.